CMakeLists.txt

```diff
@@ -72,6 +81,34 @@ set(run_traceability OFF)
 set(run_perf_tests OFF)
 set(run_reals_check OFF)
 
+if ((WIN32) AND (${GBALLOC_LL_TYPE} STREQUAL "JEMALLOC"))
+    if (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/vcpkg/scripts/buildsystems/vcpkg.cmake)
+        set(VCPKG_OVERLAY_TRIPLETS "${CMAKE_CURRENT_LIST_DIR}/triplets")
+        if(${fsanitize})
+            set(VCPKG_TARGET_TRIPLET "x64-windows-static-cpal-asan")
+        else()
+            set(VCPKG_TARGET_TRIPLET "x64-windows-static-cpal")
+        endif()
+
+        include(deps/vcpkg/scripts/buildsystems/vcpkg.cmake)
+    endif()
+
+    if ((WIN32) AND (${GBALLOC_LL_TYPE} STREQUAL "JEMALLOC"))
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules (JEMALLOC jemalloc)
+
+        pkg_search_module(JEMALLOC REQUIRED jemalloc)
+        include_directories(${JEMALLOC_INCLUDE_DIRS})
+
+        #this wraps the jemalloc library in a target so it can be used in the rest of the build easily
+        add_library(jemalloc INTERFACE)
+        target_link_libraries(jemalloc INTERFACE ${pkgcfg_lib_JEMALLOC_jemalloc_s})

```

Comment:
>pkgcfg_lib_JEMALLOC_jemalloc_s [](http://example.com/codeflow?start=51&length=30)

There's a level of "magic" concerning this name here, like ... where is it coming from and how was it discovered.

Would be nice to have some comments that explain that, or links to documentation. What if *I* need to maintain this in 2 months?

Reply:
Added comments